### ISSUE TITLE
voidptr println problem

### ISSUE THREAD

**Issue by kitech** [vlang/v#3122]
**V version:** V 0.1.23
**OS:** linux

**What did you do?**
```
fn main() {
	ptr := voidptr(5)
	println('ptr=$ptr')
	println(ptr)
}
```

**What did you expect to see?**
```
ptr=0x5 
0x5
```
 
**What did you see instead?**
```
unhandled sprintf format "voidptr"
`voidptr` needs to have method `str() string` to be printable
```

**Comment by Kamori** [comment]
Bug or not, here is a workaround.

```v
fn main() {
	ptr := voidptr(5)
	printf('ptr=%#x', ptr)

}
```

output
`ptr=0x5`

**Comment by kitech** [comment]
Thanks, will use this alternative method.
Then wait a confirm that is this need support in `println`.

**Comment by lcddh** [comment]
```
	ptr := voidptr(5)
	print(int(ptr))
```

**Comment by spytheman** [comment]
As a workaround, there is also `println( ptr_str( ptr ) )`

**Comment by spytheman** [comment]
@Kamori , printf does not work on windows unfortunately :-| .

**Comment by danieldaeschle** [comment]
```
cgen error: could not generate string method for type 'voidptr'
```

**Comment by ka-weihe** [comment]
Fixed in #4698
